sdk/iot/hub/src/az_iot_hub_client.c

```diff
@@ -13,6 +13,8 @@
 static const uint8_t hub_client_forward_slash = '/';
 static const uint8_t hub_client_param_separator = '&';
 static const uint8_t hub_client_param_equals = '=';
```

Comment:
why have these defined twice

static const uint8_t hub_client_param_separator = '&';
static const uint8_t hub_client_param_equals = '=';	static const uint8_t hub_client_param_equals = '=';

U can just use az_span_ptr(hub_client_param_separator_span) and only have one definition?

Reply:
you are right good call. updated